webpack.common.js

```diff
@@ -52,7 +52,11 @@ module.exports = {
       },
       {
         test: /\.(ttf|woff|eot|svg|png|woff2|gif|jpg)(\?[\s\S]+)?$/,
-        loader: 'url-loader?limit=100000'
+        loader: 'url-loader',
+        options: {
+            'limit': 10000,
+            'name': 'assets/[hash].[ext]'
+        }
```

Comment:
Same comment as #3: We should check if files that are not inlined are able to be loaded by the app when the JS bundle is loaded from a CDN.